### DISCUSSION TITLE
Hide archived users: Use the same view options for users as in collections that have soft-delete (archive) enabled

### DISCUSSION THREAD

**Discussion by vanling** [directus/directus#17607] [Feature Requests]
### Summary

I like to be able to hide archived users. Basicly it's the same idea as in collections that have soft-delete (archive) enabled.

### Motivation

I'm unable to delete some older users because of foreign key constraints, which is actually a good thing. So i am now changing the account status of those users to `archived` and filter them out via filter on top of page.

### Detailed Design

So it would be nice if 
<img width="238" alt="image" src="https://user-images.githubusercontent.com/50699657/220856440-a91c78fc-7519-43a0-b5d2-e8b76134387b.png">
is also possible on the users list.

### Must Have:
- archived filter

### Drawbacks

"whether the proposed feature can be implemented in user space"
> yes, but could be avoided by doing this and reusing existing logic

### Adoption Strategy

Would not affect anything but the view

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
